code-challenges.test.js

```diff
@@ -10,25 +10,62 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+// a describe method that lists the name of the function OR naming of the particular test.
+// describe("hello", () => {
+//   it("returns a string that says hi", () => {
+//     expect(hello()).toEqual("hi")
+//   })
+// })
+
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
 // a) Create a test with an expect statement using the variable provided.
 
+// Create a function called theGuide that takes an object and returns each string in that object with capitals applied.
+// Input is an array of objects
+// Output is same array but capitalization is correctly applied to the names
+
+describe("theGuide", () => {
+  it("takes in an array of objects and returns an array with a sentence about each with their name capitalized", () => {
+    expect(theGuide(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  })
+})
+
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
+
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
 
 // b) Create the function that makes the test pass.
 
+const theGuide = () =>{
+  let name = people.name.split('').toUpperCase() + people.name.slice(1)
+  return name.map(theGuide).join('')
+}
+
+// I know it doesn't pass the test, can I please get a pointer in where to go? I didn't really get much from cohort-mates this time on this one
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
 // a) Create a test with an expect statement using the variables provided.
 
+// Make a function called podgeFilter
+//Input is a mixed array
+//Filters out just the numbers
+// Divides numbers by 3
+//Outputs array of remainders
+
+describe("podgeFilter", () => {
+  it("returns an array of only the REMAINDERS of the numbers when divided by 3", () => {
+    expect(podgeFilter(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    expect(podgeFilter(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
+
```

Comment:
Please remember to leave if the test pass or fail after the initial run of the test and after you create a function